mobile/examples/speech_recognition/ios/SpeechRecognition/SpeechRecognizer.swift

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import Foundation
+
+private let kTokens = [
+  "<s>", "<pad>", "</s>", "<unk>", "|", "E", "T", "A", "O", "N", "I", "H", "S", "R", "D", "L", "U", "M", "W", "C", "F",
+  "G", "Y", "P", "B", "V", "K", "'", "X", "J", "Q", "Z",
+]
+
```

Comment:
Comment explaining what these are would help. Is there existing documentation somewhere we could link to that explains the various values and how they're used?

Reply:
added a comment but couldn't find a good documentation link. linked to the source example though.